sql/protocol_classic.cc

```diff
@@ -1410,6 +1410,12 @@ int Protocol_classic::read_packet() {
   }
 
   bad_packet = true;
+  /*
+    XXX: This place seems errorneous: why we keep thread for unusable
```

Comment:
s/errorneous/erroneous

Please write more descriptive comment. This one is hard to understand without investigating what is the context (we keep thread... which thread? what keep, etc...). Probably describing better what happens to the caller now and what would happen if it was `NET_ERROR_SOCKET_RECOVERABLE` would be understood easier.